src/test/java/com/yahoo/bullet/storm/JoinBoltTest.java

```diff
@@ -220,27 +216,15 @@ public void testQueryExpiry() {
     }
 
     @Test
-    public void testFailJoiningForNoQuery() {
```

Comment:
Is this test removed? We still need it right?

Reply:
Data arriving for a query that has not yet arrived is tested in "testFailJoiningForNoQueryOrReturn" - which I renamed to "testFailJoiningForNoQuery".